### ISSUE TITLE
TypeError: Property object of MemberExpression expected node to be of a type ["Expression","Super"] but instead got "TsConstAssertion"

### ISSUE THREAD

**Issue by lencioni** [coderaiser/swc-to-babel#14]
When parsing the following ts code with swc (options are `{ syntax: 'typescript' }`) and then passing the result to swc-to-babel:

```ts
const foo = ([1, 2] as const);
```

I get the following error:

```
TypeError: Property object of MemberExpression expected node to be of a type ["Expression","Super"] but instead got "TsConstAssertion"
```

Here's the relevant part of the stack trace:

```
      at Object.validate (node_modules/@babel/types/src/definitions/utils.ts:138:11)
      at validate (node_modules/@babel/types/src/validators/validate.ts:32:9)
      at validateField (node_modules/@babel/types/src/validators/validate.ts:19:3)
      at NodePath._replaceWith (node_modules/@babel/traverse/lib/path/replacement.js:169:5)
      at NodePath.replaceWith (node_modules/@babel/traverse/lib/path/replacement.js:151:8)
      at Object.<anonymous>.module.exports.convertParenthesisExpression (node_modules/swc-to-babel/lib/swc/index.js:132:10)
      at enter (node_modules/swc-to-babel/lib/swc-to-babel.js:119:24)
      at NodePath._call (node_modules/@babel/traverse/lib/path/context.js:53:20)
      at NodePath.call (node_modules/@babel/traverse/lib/path/context.js:36:14)
      at NodePath.visit (node_modules/@babel/traverse/lib/path/context.js:100:31)
      at TraversalContext.visitQueue (node_modules/@babel/traverse/lib/context.js:105:16)
      at TraversalContext.visitSingle (node_modules/@babel/traverse/lib/context.js:79:19)
      at TraversalContext.visit (node_modules/@babel/traverse/lib/context.js:133:19)
      at traverseNode (node_modules/@babel/traverse/lib/traverse-node.js:24:17)
      at NodePath.visit (node_modules/@babel/traverse/lib/path/context.js:107:52)
      at TraversalContext.visitQueue (node_modules/@babel/traverse/lib/context.js:105:16)
      at TraversalContext.visitSingle (node_modules/@babel/traverse/lib/context.js:79:19)
      at TraversalContext.visit (node_modules/@babel/traverse/lib/context.js:133:19)
      at traverseNode (node_modules/@babel/traverse/lib/traverse-node.js:24:17)
      at NodePath.visit (node_modules/@babel/traverse/lib/path/context.js:107:52)
      at TraversalContext.visitQueue (node_modules/@babel/traverse/lib/context.js:105:16)
      at TraversalContext.visitSingle (node_modules/@babel/traverse/lib/context.js:79:19)
      at TraversalContext.visit (node_modules/@babel/traverse/lib/context.js:133:19)
      at traverseNode (node_modules/@babel/traverse/lib/traverse-node.js:24:17)
      at NodePath.visit (node_modules/@babel/traverse/lib/path/context.js:107:52)
      at TraversalContext.visitQueue (node_modules/@babel/traverse/lib/context.js:105:16)
      at TraversalContext.visitMultiple (node_modules/@babel/traverse/lib/context.js:74:17)
      at TraversalContext.visit (node_modules/@babel/traverse/lib/context.js:131:19)
      at traverseNode (node_modules/@babel/traverse/lib/traverse-node.js:24:17)
      at NodePath.visit (node_modules/@babel/traverse/lib/path/context.js:107:52)
      at TraversalContext.visitQueue (node_modules/@babel/traverse/lib/context.js:105:16)
      at TraversalContext.visitQueue (node_modules/@babel/traverse/lib/context.js:111:21)
      at TraversalContext.visitSingle (node_modules/@babel/traverse/lib/context.js:79:19)
      at TraversalContext.visit (node_modules/@babel/traverse/lib/context.js:133:19)
      at traverseNode (node_modules/@babel/traverse/lib/traverse-node.js:24:17)
      at NodePath.visit (node_modules/@babel/traverse/lib/path/context.js:107:52)
      at TraversalContext.visitQueue (node_modules/@babel/traverse/lib/context.js:105:16)
      at TraversalContext.visitMultiple (node_modules/@babel/traverse/lib/context.js:74:17)
      at TraversalContext.visit (node_modules/@babel/traverse/lib/context.js:131:19)
      at traverseNode (node_modules/@babel/traverse/lib/traverse-node.js:24:17)
      at NodePath.visit (node_modules/@babel/traverse/lib/path/context.js:107:52)
      at TraversalContext.visitQueue (node_modules/@babel/traverse/lib/context.js:105:16)
      at TraversalContext.visitSingle (node_modules/@babel/traverse/lib/context.js:79:19)
      at TraversalContext.visit (node_modules/@babel/traverse/lib/context.js:133:19)
      at traverseNode (node_modules/@babel/traverse/lib/traverse-node.js:24:17)
      at NodePath.visit (node_modules/@babel/traverse/lib/path/context.js:107:52)
      at TraversalContext.visitQueue (node_modules/@babel/traverse/lib/context.js:105:16)
      at TraversalContext.visitMultiple (node_modules/@babel/traverse/lib/context.js:74:17)
      at TraversalContext.visit (node_modules/@babel/traverse/lib/context.js:131:19)
      at traverseNode (node_modules/@babel/traverse/lib/traverse-node.js:24:17)
      at NodePath.visit (node_modules/@babel/traverse/lib/path/context.js:107:52)
      at TraversalContext.visitQueue (node_modules/@babel/traverse/lib/context.js:105:16)
      at TraversalContext.visitSingle (node_modules/@babel/traverse/lib/context.js:79:19)
      at TraversalContext.visit (node_modules/@babel/traverse/lib/context.js:133:19)
      at traverseNode (node_modules/@babel/traverse/lib/traverse-node.js:24:17)
      at traverse (node_modules/@babel/traverse/lib/index.js:62:34)
      at toBabel (node_modules/swc-to-babel/lib/swc-to-babel.js:52:5)
```

I believe this is due to the const assertion (`as const`) being surrounded by parens. I don't get the error if I remove the parens:

```ts
const foo = [1, 2] as const;
```

**Comment by coderaiser** [comment]
Are you willing for a PR?

**Comment by lencioni** [comment]
@coderaiser I've started a draft PR here https://github.com/coderaiser/swc-to-babel/pull/15 but it is not quite right and I'm not sure if it is even the right way to do this. Can you take a look and advise please?

**Comment by coderaiser** [comment]
Landed in [v1.26.0](https://github.com/coderaiser/swc-to-babel/releases/tag/v1.26.0) 🎉 !
Is it works for you?
